.travis.yml

```diff
@@ -22,8 +22,10 @@ env:
   matrix:
   - LINT_CHECK="1"
   - TRANSIFEX="1"
-  - TESTS="1" ODOO_REPO="odoo/odoo"
-  - TESTS="1" ODOO_REPO="OCA/OCB"
+  - TESTS="1" ODOO_REPO="odoo/odoo" INCLUDE=partner_multi_relation
```

Comment:
Oh, I remember. It was a way to avoid some kind of side effects on other modules tests. Let's remove it and see if it works